tests/std/tests/P0896R4_ranges_alg_move/test.cpp

```diff
@@ -0,0 +1,78 @@
+#include <algorithm>
+#include <cassert>
+#include <concepts>
+#include <ranges>
+#include <utility>
+#include <range_algorithm_support.hpp>
+
+struct int_wrapper {
+    int val = 10;
+    constexpr int_wrapper() = default;
+    constexpr int_wrapper(int x) : val{x} {}
+    constexpr int_wrapper(int_wrapper&& that) : val{std::exchange(that.val, -1)} {}
+    constexpr int_wrapper& operator=(int_wrapper&& that) {
+        val = std::exchange(that.val, -1);
+        return *this;
+    }
+};
+
+constexpr void smoke_test() {
+    using ranges::move, ranges::move_result, ranges::iterator_t;
+    using std::same_as;
+
+    // Validate that move_result aliases in_out_result
+    STATIC_ASSERT(same_as<move_result<int, double>, ranges::in_out_result<int, double>>);
+
+    // Validate dangling story
+    STATIC_ASSERT(same_as<decltype(move(borrowed<false>{}, static_cast<int*>(nullptr))), move_result<ranges::dangling, int*>>);
+    STATIC_ASSERT(same_as<decltype(move(borrowed<true>{}, static_cast<int*>(nullptr))), move_result<int*, int*>>);
+
+    int const input[] = {13, 53, 12435};
+    {
+        int output[] = {-1, -1, -1};
+        auto result = move(move_only_range{input}, move_only_range{output}.begin());

```

Comment:
I am unsure what happens here, as you pulled in std::move and std::ranges::move.

Could you please qualify this call?

Reply:
> as you pulled in std::move

Where?

Reply:
Blind me. I was missing the std:: in front of ranges and assume all test files have `using namespace std`.

Sorry for the noise